scripts/class/Hero.js

```diff
@@ -5,6 +5,20 @@ class Hero {
     */
     constructor(level){
         //TODO generate initial hero/random monster with random stats
+        this.name = '';
+        this.experience = 0;
+        this.level = level;
+        this.stats = new Stats();
+        this.avatar = ''; //URI to hero image
```

Comment:
Again, this should be working. Find a default image (or placeholder image) and link it here